js/views/inbox_view.js

```diff
@@ -2,282 +2,282 @@
  * vim: ts=4:sw=4:expandtab
  */
 (function () {
-    'use strict';
```

Comment:
Seems like it's not the best idea to eliminate this unless you know something I don't. We're not passing this code into webpack, which would add the `'use strict'` for us...

Reply:
Good catch! Turns out this was because of the ESLint `strict` rule and the default setting

```
  "parserOptions": {
    "sourceType": "module"
  }
```

For our existing `js/*.js` files I changed it to:

```
  "parserOptions": {
    "sourceType": "script"
  }
```

and now `'use strict';` is correctly required. Thank you! 🙇